next.config.js

```diff
@@ -0,0 +1,7 @@
+const withPWA = require('next-pwa')
+
+module.exports = withPWA({
+    pwa: {
```

Comment:
What indentation is this @RajvirSingh1313 2 or 4?

Reply:
at last line, one single space after the bracket

Reply:
Done!